api/pacman-api-compliance/src/main/java/com/tmobile/pacman/api/compliance/repository/ComplianceRepositoryImpl.java

```diff
@@ -2230,7 +2232,8 @@ private StringBuilder formatURL(String assetGroup, String resourcetype, String a
         }
         return urlToQuery;
     }
-    private  String createAuditTrail(String ds, String type, String status, String id,String createdBy, String _type, Map<String, Object> parentDetMap, String target) {
+
+    private String createAuditTrail(String ds, String type, String status, String id, String createdBy, String _type, Map<String, Object> parentDetMap, String target, Map<String, String> optionalAuditFields) {

```

Comment:
please extract the params to a CreateAuditTrail model, this param list is getting out of hand and if it keeps growing will be unmanageable.

Reply:
done